tools/coqdep/lib/dep_info.ml

```diff
@@ -8,12 +8,16 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-module Dynlink = struct
-  type t = Opt | Byte | Both | No | Variable
+module Dep = struct
```

Comment:
Weird that git picked this up as a file renaming. Can you fix that?

Reply:
Same for makefile.mli

Reply:
How can we fix this?

As far as I can tell git doesn't track renames, it is `git diff` who infers them after the fact. But maybe my git knowledge is obsolete.

Using two commits here seems overkill.

Reply:
Fair enough, hopefully the history won't be weird.